src/main/java/com/amplitude/Amplitude.java

```diff
@@ -174,6 +174,7 @@ public synchronized void flushEvents() {
     if (eventsToSend.size() > 0) {
       List<Event> eventsInTransit = new ArrayList<>(eventsToSend);
       eventsToSend.clear();
+      logger.log(logger.getTimeStamp(), eventsInTransit.size() + " events to flush");
```

Comment:
I hope the earlier method was called debug other than log, two log method is a bit confusing.